packages/mux-player/package.json

```diff
@@ -14,7 +14,8 @@
   "license": "MIT",
   "scripts": {
     "lint": "eslint src/ --ext .js,.jsx,.ts,.tsx",
-    "test": "web-test-runner **/*test.js --coverage --config test/web-test-runner.config.js",
+    "test": "web-test-runner **/*test.js --port 8001 --coverage --config test/web-test-runner.config.js",
```

Comment:
non-blocking: assuming this is because they run in parallel on the github action. This'll also be a pain to maintain over time. I suspect we should be able to do something clever in the web-test-runner.config file(s) to detect and/or otherwise programmatically assign port numbers for our packages.